src/troute-network/troute/hyfeature_network_utilities.py

```diff
@@ -60,7 +64,7 @@ def build_da_sets(da_params, run_sets, t0):
     if reservoir_persistence_da:
         usgs_da = reservoir_persistence_da.get('reservoir_persistence_usgs', False)
         usace_da = reservoir_persistence_da.get('reservoir_persistence_usace', False)
-        canada_da = reservoir_persistence_da.get('reservoir_persistence_canada', False)
+        canada_da = reservoir_persistence_da.get('reservoir_persistence_greatLake', False)
```

Comment:
Maybe change `canada_da` to `GreatLakes_da`?

Reply:
Done